ddl/ddl_api.go

```diff
@@ -5476,6 +5476,13 @@ func extractTblInfos(is infoschema.InfoSchema, oldIdent, newIdent ast.Ident, isA
 		// oldIdent is equal to newIdent, do nothing
 		return nil, 0, nil
 	}
+	//View can be renamed only in the same schema. Compatible with mysql
+	if is.TableIsView(oldIdent.Schema, oldIdent.Name) {
+		if oldIdent.Schema != newIdent.Schema {
```

Comment:
```
if is.TableIsView(oldIdent.Schema, oldIdent.Name) && oldIdent.Schema != newIdent.Schema {
```

Reply:
I think maybe it is more semantics to seprate two `if statement`
1.We judege the table is view or not firstly. And If it is, we continue to do other operations.
2.If there are other operations about `view table` in the future. We can add code directly below `if is.TableIsView(oldIdent.Schema, oldIdent.Name) {`